setup.py

```diff
@@ -9,7 +9,10 @@
     requirements = requirements_file.readlines()
 
 with open('requirements_dev.txt') as requirements_file:
-    test_requirements = requirements_file.readlines() + requirements
+    test_requirements = [
+        req for req in requirements_file.readlines()
+        if 'git+' not in req
+    ] + requirements
```

Comment:
Yup. There was some discussion on it that didn't seem promising. Let me find it...